setup.py

```diff
@@ -24,7 +24,6 @@
         'tensor2tensor/bin/t2t-datagen',
         'tensor2tensor/bin/t2t-decoder',
         'tensor2tensor/bin/t2t-make-tf-configs',
-        'tensor2tensor/bin/t2t-bleu',
```

Comment:
Is the deletion of t2t-bleu intentional?
(I can understand it, if yes.)

Reply:
Thanks for noticing that. No, it was not intentional and likely due to a bad internal merge of the PR.